libraries/AP_NavEKF3/AP_NavEKF3_core.h

```diff
@@ -581,6 +588,10 @@ class NavEKF3_core : public NavEKF_core_common
         bool corrected;             // true when the velocity has been corrected for sensor position
     };
 
+    struct drag_elements : EKF_obs_element_t {
```

Comment:
I wonder why we have a structure with just one element in it..

Reply:
Commonality with the other observation methods (we are using the same buffer write and retrieval methods.